app/src/main/java/org/kiwix/kiwixmobile/main/KiwixMainActivity.kt

```diff
@@ -116,17 +114,17 @@ class KiwixMainActivity : CoreMainActivity() {
   }
 
   private fun closeReaderDrawer() {
-    navigation_container.closeDrawer(reader_drawer_nav_view)
+    drawerContainerLayout.closeDrawer(reader_drawer_nav_view)
   }
 
   private fun readerDrawerIsOpen() =
-    navigation_container.isDrawerOpen(reader_drawer_nav_view)
+    drawerContainerLayout.isDrawerOpen(reader_drawer_nav_view)
 
   override fun navigationDrawerIsOpen(): Boolean =
```

Comment:
these don't seem like they should be overrides if the parent has the field now

Reply:
true! nice.